### PR TITLE
Remove CfA, add make donation section

### PR DESCRIPTION
**Pull request by Lucie-H**

### CODE REVIEW

File: templates/pages/index.html.njk [hunk 1]
```diff
@@ -34,14 +34,13 @@
   <section class="c-content">
     <div class="l-container">
       <h2 class="fz--beta">
-        Call for Artists is open!
+        Make a scholarship donation!
```
Comment (lukaszklis) [inline templates/pages/index.html.njk]: Don’t we want to apply the same titlecase rule as for other headlines here?

Comment (Kriesse) [reply templates/pages/index.html.njk]: Yup I'd use titlecase too
`Make a Scholarship Donation`

File: templates/_macros.njk [hunk 2]
```diff
@@ -81,13 +81,13 @@
...
 
-{% macro callForArtistsButton(link, text = '', truncateOnMobile = false, openNewTab = true, centered = true) %}
+{% macro callForDonationsButton(link, text = '', truncateOnMobile = false, openNewTab = true, centered = true) %}
   <p {% if centered %}class="btn-block"{% endif %}>
     <a href="{{ link }}" class="c-btn" {% if openNewTab %}target="_blank"{% endif %}>
       {% if text %}
         {{ text }}
       {% else %}
-        Submit an Art
+        🙏 Donate now
```
Comment (Kriesse) [inline templates/_macros.njk]: titlecase = Donate Now

**Comment by Lucie-H** [comment]
@Kriesse done!

**Pull request merged**